starter-code/exBook/views/place/show.ejs

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Google Maps | Express</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+</head>
+
+<body>
+    <h3 id="restaurantName" data-id="<%= restaurant.id %>">
+        <%= restaurant.name %>
+    </h3>
+    <p>
+        <%= restaurant.description %>
+    </p>
+    <a href="/">List</a>
+        <div id="map"></div>
+
+</body>
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCbBJPvYIaR5ozIx_0-uWiGeO9cK4IxHww">
+
+</script>
+<script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+    crossorigin="anonymous"></script>
+<script type="text/javascript">
+    const myPlace = <%- JSON.stringify(restaurant) %>
```

Comment:
You are defining a `myPlace` variable but `main.js` is using the `myPlaces` variable.